install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1772,7 +1772,17 @@ loadBalancer:
   # mode: snat
 
   # -- acceleration is the option to accelerate service handling via XDP
-  # e.g. native, disabled
+  # disabled - do not use XDP
+  # native - XDP BPF program is run directly out of the networking driver’s
+  #   early receive path.
+  # generic - For drivers not implementing native or offloaded XDP yet, the
+  #   kernel provides an option for generic XDP which does not require any
+  #   driver changes since run at a much later point out of the networking
+  #   stack. This setting is primarily targeted at developers who want to
+  #   write and test programs against the kernel’s XDP API, and will not
+  #   operate at the performance rate of the native or offloaded modes.
+  # best-effort - Use native where possible, but ignore devices without XDP
```

Comment:
"ignore devices" can be a bit misleading. Cilium will still handle traffic to/from this device using the program attached to a TC filter. Perhaps something like "Use native mode XDP acceleration on devices that support it".